tools/enable_vmx.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
```

Comment:
nit: Add a comment explaining what this script does? (For folks unfamiliar with instance templates)

Reply:
Done.